### PR TITLE
Create Erik-Dennis.md

### PR DESCRIPTION
**Pull request by Erik-Dennis**
## Describe your project in 1-3 sentences

 I am building a smart home controller with a touchscreen interface

## Hack Club Slack handle

Erik Dennis

## Checklist before submitting proposal:

- [Y] I am a current high school student (or younger).
- [Y] I have submitted an RSVP at <https://hackclub.com/winter>
- [Y] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [Y] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @Erik-Dennis this is super cool, thanks for submitting a PR! i'm wondering, what will you be learning from this process, how long do you think it'll take for you to set this up, and will you be writing any code?

thank you sm!

**Comment by Erik-Dennis** [comment]
I'll be learning about device integration and networking. I wasn't planning on writing code for it, but I might extend the project and create a custom HTML webpage for the dashboard and look into using API's. The integration through home assistant would probably take 1-3 days depending on how much free time I have, the customization with the webpage would be an extension that would go over the 10 days and probably take a couple months on top school.

File: Erik-Dennis.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+name: @Erik-Dennis
+project: Smart Device Controller
+---
+
+# Project Name
+
+Smart Device Controller
```
Comment (bellesea) [inline Erik-Dennis.md:8]:
```suggestion
```

File: Erik-Dennis.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+name: @Erik-Dennis
+project: Smart Device Controller
+---
+
+# Project Name
+
```
Comment (bellesea) [inline Erik-Dennis.md:7]:
```suggestion
```

File: Erik-Dennis.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+name: @Erik-Dennis
+project: Smart Device Controller
+---
+
+# Project Name
```
Comment (bellesea) [inline Erik-Dennis.md:6]:
```suggestion
# Smart Device Controller
```

**Comment by bellesea** [comment]
> I'll be learning about device integration and networking. I wasn't planning on writing code for it, but I might extend the project and create a custom HTML webpage for the dashboard and look into using API's. The integration through home assistant would probably take 1-3 days depending on how much free time I have, the customization with the webpage would be an extension that would go over the 10 days and probably take a couple months on top school.

ahhh cool! thank you for the rapid response :) what we're looking for in project is something you spend a significant amount of time on, that you can add something to that makes it your own, and that you can learn from. do you think that this fits the criteria?

also, do you need to purchase an SD card and power supply compatible w/ the pi or do you already have that?

thank you sm!

**Comment by Erik-Dennis** [comment]
I have a USB C charger I can add to it, it's not the official but it should work fine. I also have an SD card.

**Comment by Erik-Dennis** [comment]
I think the end result meets that criteria. I haven't ever used home assistant but it'll be a learning process. The webpage is the end result I'm going for but like stated above that'll go over the 10 days.

**Comment by bellesea** [comment]
> I think the end result meets that criteria. I haven't ever used home assistant but it'll be a learning process. The webpage is the end result I'm going for but like stated above that'll go over the 10 days.

sounds good, @Erik-Dennis could you please update your file with plans for the webpage? it's ok that it'll go over 10 days :) thank you sm!

**Comment by Erik-Dennis** [comment]
Will do.

**Comment by bellesea** [comment]
> Will do.

thanks for pushing the changes! btw, how do you plan on programming your pi 4? do you have a way to program the SD card or will you be connecting a monitor etc.?

**Comment by Erik-Dennis** [comment]
Yeah I'll probably use Rufus or pi installer

**Review by bellesea — APPROVED** [review]
@Erik-Dennis can't wait to see you build this! maybe we could even control the lights in your room remotely hehe, that would be such a cool and fun zoom call!

a fabulous member of @hackclub/bank-ops will be reaching out with next steps :)

**Review by MelSmith104 — COMMENTED** [review]
hey @Erik-Dennis love your project idea! a smart controller would be so useful to have at the house! im dm'ing you on slack so that we can chat about getting you set up on Bank for the grant!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**